### PR TITLE
Update README.md with usage and known issues.

### PR DESCRIPTION
**Pull request by dlorenc**
You can see the README.md here: https://github.com/dlorenc/minikube/tree/readme

### CODE REVIEW

**Comment by luxas** [comment]
Be aware that you have two sections that explains nearly the same thing: https://github.com/dlorenc/minikube/blob/readme/README.md#usage; that's running `minikube` only, and https://github.com/dlorenc/minikube/blob/readme/README.md#run-instructions that's running `out/minikube`

Consider merging them or something

**Comment by aaron-prindle** [comment]
I think that the usage, features, and known issues section are good.  As @luxas mentioned though, we should probably try to keep the usage and run instructions section consistent regarding using:  minikube  OR  ./out/minikube start.  Or we could perhaps merge them as I don't think the 'kubectl config' commands and without them the Run Instructions section is very similar to the Usage section.

**Comment by dlorenc** [comment]
Thanks for the review. I reorganized it a bit so the ./out/minikube is only mentioned in the development instructions, the "usage" section is now meant for end users who will have minikube on their path.

File: README.md [hunk 1]
```diff
@@ -10,6 +10,55 @@ Minikube is a tool that makes it easy to run Kubernetes locally. Minikube runs
...
+```shell
+$ minikube start
+Starting local Kubernetes cluster...
+Running pre-create checks...
+Creating machine...
+Starting local Kubernetes cluster...
+Kubernetes is available at https://192.168.99.100:443.
+
+$ $(minikube docker-env)
+$ docker build -t helloworld .
```
Comment (luxas) [inline README.md]: As a user, I would wonder what this is... Is it a built-in thing or is it some required step or is it just a demo... and how may I try that demo out in that case...

File: README.md [hunk 2]
```diff
@@ -10,6 +10,55 @@ Minikube is a tool that makes it easy to run Kubernetes locally. Minikube runs
...
+
+Here's a brief overview of minikube usage:
+
+```shell
+$ minikube start
+Starting local Kubernetes cluster...
+Running pre-create checks...
+Creating machine...
+Starting local Kubernetes cluster...
+Kubernetes is available at https://192.168.99.100:443.
```
Comment (luxas) [inline README.md]: Is this a constant or a dynamic IP? The user maybe wants to know that

File: README.md [hunk 3]
```diff
@@ -10,6 +10,55 @@ Minikube is a tool that makes it easy to run Kubernetes locally. Minikube runs
...
+  * Advanced scheduling policies
+  * DaemonSets
+ * Alternate runtimes, like rkt.
+
+If you need these features, don't worry! We're planning to add these to minikube over time. Please leave a note in the
+issue tracker about how you'd like to use minikube!
+
 ## Design
 
 Minikube uses [libmachine](https://github.com/docker/machine/tree/master/libmachine) for provisioning VMs, and [localkube](https://github.com/redspread/localkube)
```
Comment (luxas) [inline README.md]: While you're at it, we're not using Redspread's `localkube` anymore, maybe we should mention they donated it for continued development in this repo?

Comment (dlorenc) [inline README.md]: Good idea. Done.

File: README.md (hunk 2 above)
Comment (dlorenc) [inline README.md]: clarified.

File: README.md (hunk 1 above)
Comment (dlorenc) [inline README.md]: Clarified above that this is just a demo, and added a link to the tutorial.

**Comment by luxas** [comment]
LGTM

**Pull request merged**